images/diagnostics/scripts/openshift-network-debug

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# This script is used by network diagnostics.
+# Based on containerized/non-containerized openshift install,
+# it sets the environment so that docker, openshift, iptables, etc.
+# binaries are availble for network diagnostics.
+#
+set -o nounset
+set -o pipefail
+
+node_rootfs=$1
+shift
+cmd="${@}"
+node_image="openshift/node"
```

Comment:
Instead of passing the node image via config, we can do pattern match of node image. '^openshift.*/node$' satisfies future versions?